Sources/SolanaSwift/Deprecated/SolanaSDK+Actions/SolanaSDK+Send.swift

```diff
@@ -52,7 +52,7 @@ extension SolanaSDK {
                 .flatMap { [weak self] in
                     guard let self = self else {return .error(SolanaError.unknown)}
                     // form instruction
-                    let instruction = SystemProgram.transferInstruction(
+                    let instruction = SystemProgram().transferInstruction(
```

Comment:
Why do we initialise SystemProgram? Shouldn't it be static function?

Reply:
I have done this before, but static functions may be hard to be tested. We can't make multiple instance to test function separately

Reply:
We can use [static function in protocol](https://medium.com/openclassrooms-product-design-and-engineering/injecting-and-mocking-static-frameworks-in-swift-b4fc410ab3ae).

Or using [mock library](https://github.com/Brightify/Cuckoo).

Programs are unique entity and represent an hard-coded template. It's doesn't make sense to change ProgramID or something else in transfer instruction. There should be only one single SystemProgram.

I think it maybe a special case where we don't need protocol here.

Reply:
Ok. So I will omit the protocols. Thanks (y)